unittests/InstructionCountCI/FlagM/x87_f64.json

```diff
@@ -4198,7 +4198,8 @@
         "mov v2.16b, v0.16b",
         "str d2, [x4]",
         "mov x21, v2.d[1]",
-        "strh w21, [x4, #8]",
+        "add x22, x4, #0x8 (8)",
+        "strh w21, [x22]",
```

Comment:
Why is this one (and others in other files) still not appending the immediate to the store instruction?

Reply:
Oops. 64-bit test, thought I was looking at 32-bit test.